tests/python_package_test/test_utilities.py

```diff
@@ -97,3 +99,15 @@ def dummy_metric(_, __):
                 actual_log_wo_gpu_stuff.append(line)
 
     assert "\n".join(actual_log_wo_gpu_stuff) == expected_log
+
+
+def test_smoke_custom_parser(tmp_path):
```

Comment:
This file is used to test helper public functions from Python-package (https://lightgbm.readthedocs.io/en/latest/Python-API.html#utilities).
Please move this test into
https://github.com/microsoft/LightGBM/blob/master/tests/python_package_test/test_basic.py
as it checks core functionality of LightGBM.

Reply:
Done.